types/react-bootstrap/lib/NavItem.d.ts

```diff
@@ -2,7 +2,7 @@ import * as React from "react";
 import { SelectCallback, Sizes } from "react-bootstrap";
 
 declare namespace NavItem {
-    export interface NavItemProps extends React.HTMLProps<NavItem> {
+    export interface NavItemProps extends Omit<React.HTMLProps<NavItem>, "onToggle"> {
```

Comment:
IIRC extending from Omit is cursed, but I guess it's not uncommon on DT.

Reply:
Yeah, I suspect as much since we're going from an `interface` to a `type`. But I'm not aware how to use this pattern in TypeScript without `Omit` i.e. I don't know how to override a property without `Omit`. TypeScript always wants properties to be assignable when extending an `interface`